Installation/cmake/modules/CGAL_CreateSingleSourceCGALProgram.cmake

```diff
@@ -18,6 +18,18 @@ function(create_single_source_cgal_program firstfile )
 
     add_executable(${exe_name} ${all})
 
+    if(BUILD_TESTING)
+      if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/${exe_name}.cin")
+        set(ARGS "${CMAKE_CURRENT_SOURCE_DIR}/${exe_name}.cin")
+      elseif(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/${exe_name}.cmd")
+        file(STRINGS "${CMAKE_CURRENT_SOURCE_DIR}/${exe_name}.cmd" ARGS LIMIT_COUNT 1)
```

Comment:
Anyway, I would like to postpone those improvements for later.
Currently, the small patch I wrote has already saved my some time, for several weeks, while working for Mesh_3, or Kernel_23. I would like to integrate the current patch in CGAL-4.8 and CGAL-4.9 as fast as possible.